src/chain_spec.rs

```diff
@@ -424,26 +429,26 @@ macro_rules! staking {
     ($auths:expr, $stakers:expr, $cap:expr) => {
         pallet_staking::GenesisConfig {
             minimum_validator_count: 1,
-            validator_count: 20,
+            validator_count: 40,
             validator_commission_cap: $cap,
             stakers: $stakers,
             invulnerables: vec![],
             slash_reward_fraction: sp_runtime::Perbill::from_percent(10),
-            min_bond_threshold: 0,
+            min_bond_threshold: 0, // Will be updated to 50,000 POLYX once network has stabalised, before external operators join
```

Comment:
Have to be careful because this will break ITN rewards.

Reply:
The MinimumBondThreshold only applies to operators, not to non-operating stakers.